server/jetstream_api.go

```diff
@@ -3129,6 +3129,14 @@ func (s *Server) jsConsumerCreate(sub *subscription, c *client, a *Account, subj
 	// Make sure we have sane defaults.
 	setConsumerConfigDefaults(&req.Config)
 
+	// Check if we have a BackOff defined that MaxDeliver is within range etc.
+	if lbo := len(req.Config.BackOff); lbo > 0 && req.Config.MaxDeliver <= lbo {
+		err := errors.New("max deliver required to be > length backoff values")
```

Comment:
Remove this, and then have `resp.Error = NewJS...` that you will get adding the new error.